test/bdd/fixtures/well-known-server/docker-compose.yml

```diff
@@ -18,3 +18,10 @@ services:
       - "80:80"
     volumes:
       - ./config:/web/.well-known/did-bloc
+    networks:
```

Comment:
rename from well-known-server to discovery-server

Reply:
done